src/intentManager.js

```diff
@@ -219,70 +154,38 @@ define([
         icon: 'icon-vellum-android-intent',
         isODKOnly: true,
         isTypeChangeable: false,
-        intentTag: null,
         init: function (mug, form) {
+            mug.p.intentXmlns = mug.p.intentXmlns || DEFAULT_XMLNS;
         },
         spec: {
             androidIntentAppId: {
                 lstring: 'Intent ID',
                 visibility: 'visible',
                 widget: androidIntentAppId,
-                serialize: function (value, key, mug, data) {
```

Comment:
Were the changes in serialize/deserialize intended to preserve backward compatibility with older versions of Vellum? I'm not sure how much it matters, but just wanted to make sure you were aware in case any bug reports came in about that. This could happen if someone is saving copied questions in Excel, for example.